packages/core/interfaces/src/index.ts

```diff
@@ -1,3 +1,5 @@
+/* istanbul ignore file */
```

Comment:
I believe you could add in `collectCoverageFrom` entry:
```
'!interfaces/**/*',
```
as coverage from interfaces should not be collected at all